<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Sebastián Sánchez Cuartas" />


<title>Relationship between Evolutionary Divergence and Historical Connectivity in the Northern Andean Páramos During the Pleistocene</title>

<script src="site_libs/header-attrs-2.27/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>










<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}

.tocify-subheader {
  display: inline;
}
.tocify-subheader .tocify-item {
  font-size: 0.95em;
}

</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html"></a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Relationship between Evolutionary
Divergence and Historical Connectivity in the Northern Andean Páramos
During the Pleistocene</h1>
<h4 class="author">Sebastián Sánchez Cuartas</h4>
<h4 class="date">agosto 21, 2024</h4>

</div>


<div id="introduction" class="section level1">
<h1><strong>Introduction</strong></h1>
<p>The Andean páramos, recognized for their unique biodiversity and
specific adaptations to high mountain conditions, have been shaped by
geological and climatic events, particularly during the Pleistocene.
This period was characterized by recurrent climate fluctuations
associated with Earth’s orbit around the sun, which led to the expansion
and contraction of páramo environments, generating a complex
connectivity history (Figure 1, 2). To investigate how these climatic
fluctuations may have influenced the diversification of
páramo-associated organisms, we employed an algorithm specifically
developed to calculate historical connectivity based on a paleoclimatic
model describing the altitudinal extent of the páramo and its duration
during the Pleistocene. We used these connectivity estimates as
predictors of genetic divergence in a Generalized Dissimilarity Model
(GDM) to establish a link between diversification resulting from
continuous fragmentation during glacial cycles.</p>
<div class="figure" style="text-align: center">
<img src="flickering.gif" alt="**Figure 1.** *Shown below are the hypothetical dynamics of the páramos during the Pleistocene, resulting from climatic fluctuations. The GIF illustrate how the forest line rises or falls in response to temperature variations. This GIF was created from a video by Catalino Giraldo based on the work of Flantua et al. (2019), which is available at [this link](https://youtu.be/-Wcp18vBDK4)*" width="75%" />
<p class="caption">
<strong>Figure 1.</strong> <em>Shown below are the hypothetical dynamics
of the páramos during the Pleistocene, resulting from climatic
fluctuations. The GIF illustrate how the forest line rises or falls in
response to temperature variations. This GIF was created from a video by
Catalino Giraldo based on the work of Flantua et al. (2019), which is
available at <a href="https://youtu.be/-Wcp18vBDK4">this link</a></em>
</p>
</div>
<p>…</p>
<div class="figure" style="text-align: center">
<img src="dinmov.gif" alt="**Figure 2**. *Below are shown the possible movement dynamics of species (represented by points) during periods of higher or lower connectivity. This GIF was created from a video by Catalino Giraldo based on the work of Flantua et al. (2019) and is available at [this link](https://youtu.be/-Wcp18vBDK4)*" width="75%" />
<p class="caption">
<strong>Figure 2</strong>. <em>Below are shown the possible movement
dynamics of species (represented by points) during periods of higher or
lower connectivity. This GIF was created from a video by Catalino
Giraldo based on the work of Flantua et al. (2019) and is available at
<a href="https://youtu.be/-Wcp18vBDK4">this link</a></em>
</p>
</div>
<p>…</p>
<p>To carry out the complete process of relating historical connectivity
with evolutionary distances of páramo organisms, the following provides
a detailed flowchart of the process (Figure 1), along with direct links
to each step and its corresponding GitHub repository. This material
facilitates the reconstruction of the procedure and is applicable to
both the raw evolutionary distance data of the Espeletine subtribe used
in the original study and other groups with ecosystems in the northern
Andean páramos, whose phylogeny is an ultrametric tree with divergence
times not exceeding 2.2 million years between groups. This is because
the paleoclimatic model by Flantua et al. (2019) covers up to that
age.</p>
<p>The following steps are mandatory, and detailed information for each
step can be found in the provided links:</p>
<ol style="list-style-type: decimal">
<li><strong>Reclassifying a Digital Elevation Model in R for
Connectivity Modeling</strong> - <a
href="https://innerhaze.github.io/DEM-Reclassification-R-Paramo-Connectivity-Modeling/">See
the Code and Instructions</a> and <a
href="https://github.com/innerhaze/DEM-Reclassification-R-Paramo-Connectivity-Modeling/tree/main">Repository</a>.</li>
</ol>
<p><strong>Description</strong>: Reclassifying a digital elevation model
in R for connectivity modeling based on paleoclimatic data</p>
<ol start="2" style="list-style-type: decimal">
<li><strong>Convert Rasters to INT8 Format and Reproject to EPSG:
32618</strong> - <a
href="https://innerhaze.github.io/Convert-Rasters-to-INT8-Format-and-or-Reproject-to-EPSG-32618/">See
the Code and Instructions</a> and <a
href="https://github.com/innerhaze/Convert-Rasters-to-INT8-Format-and-or-Reproject-to-EPSG-32618">Repository</a>.</li>
</ol>
<p><strong>Description</strong>: This R script takes all raster files in
.tiff format from a folder, converts them to INT8 format, reprojects
them to the EPSG: 32618 coordinate system, and then saves them in the
output folder in the same format.</p>
<ol start="3" style="list-style-type: decimal">
<li><strong>Custom Pixel Reclassification in Raster Data Using Specified
Coordinates</strong> - <a
href="https://innerhaze.github.io/Custom-Pixel-Reclassification-in-Raster-Data-Using-Specified-Coordinates/">See
the Code and Instructions</a> and <a
href="https://github.com/innerhaze/Custom-Pixel-Reclassification-in-Raster-Data-Using-Specified-Coordinates">Repository</a>.</li>
</ol>
<p><strong>Description</strong>: This R script reclassifies pixels in
raster datasets based on coordinates from a CSV file. It assigns values
to identified pixels and allows customization of these values for
different analyses. Useful for applications like connectivity
modeling.</p>
<ol start="4" style="list-style-type: decimal">
<li><strong>Assignment of Unique Identifiers to Patches in Connectivity
Modeling Using Graphab</strong> - <a
href="https://innerhaze.github.io/Assignment-of-Unique-Identifiers-to-Patches-in-Connectivity-Modeling-Using-Graphab/">See
the Code and Instructions</a> and <a
href="https://github.com/innerhaze/Assignment-of-Unique-Identifiers-to-Patches-in-Connectivity-Modeling-Using-Graphab">Repository</a>.</li>
</ol>
<p><strong>Description</strong>: This R code assigns unique numerical
IDs to patches in Graphab to preserve species identity. It uses a CSV
file with species names and coordinates to maintain traceability and
facilitate the recovery of original identities in connectivity
analysis.</p>
<ol start="5" style="list-style-type: decimal">
<li><strong>Java-R Integration with Graphab: Cost Surface Processing and
Data Management</strong> - <a
href="https://innerhaze.github.io/Java-R-Integration-with-Graphab-Cost-Surface-Processing-and-Data-Management/">See
the Code and Instructions</a> and <a
href="https://github.com/innerhaze/Java-R-Integration-with-Graphab-Cost-Surface-Processing-and-Data-Management">Repository</a>.</li>
</ol>
<p><strong>Description</strong>: Integrates Java within R using Graphab
for automated processing of cost surfaces. Utilizes rJava to execute
Graphab .jar files and manage connectivity modeling. Includes a for loop
for handling multiple cost surfaces and renaming IDs from patches based
on species data.</p>
<ol start="6" style="list-style-type: decimal">
<li><strong>R Algorithm for Analyzing Historical Connectivity of Páramo
Lineages During the Pleistocene</strong> - <a
href="https://innerhaze.github.io/R-Algorithm-for-Historical-Connectivity-Analysis-of-Paramo-Lineages-in-the-Pleistocene/">See
the Code and Instructions</a> and <a
href="https://github.com/innerhaze/R-Algorithm-for-Historical-Connectivity-Analysis-of-Paramo-Lineages-in-the-Pleistocene">Repository</a>.</li>
</ol>
<p>Description: The algorithm calculates historical connectivity between
lineage pairs by analyzing evolutionary distances and climatic
fluctuations during the Pleistocene. Using the pollen database from
Flantua et al. (2019) with various páramo configurations, it computes a
weighted average of historical connectivity. This helps assess how
paleoclimatic conditions influenced lineage divergence.</p>
<ol start="7" style="list-style-type: decimal">
<li><strong>Alignment of Two Distance Matrices for Comparative
Analysis</strong> - <a
href="https://innerhaze.github.io/Alignment-of-Two-Distance-Matrices-for-Comparative-Analysis/">See
the Code and Instructions</a> and <a
href="https://github.com/innerhaze/Alignment-of-Two-Distance-Matrices-for-Comparative-Analysis">Repository</a>.</li>
</ol>
<p>Description: This R code aligns two distance matrices with identical
row and column names but different data. It reorders one matrix to match
the other, allowing for cohesive relational analysis. This alignment
ensures both matrices are mirrored for accurate comparative
analysis.</p>
<ol start="8" style="list-style-type: decimal">
<li><strong>GDM</strong></li>
</ol>
<div id="references" class="section level2">
<h2><strong>References</strong></h2>
<p>Flantua, S. G. A., O’Dea, A., Onstein, R. E., Hooghiemstra, H., &amp;
Hoorn, C. (2019). The flickering connectivity system of the north Andean
páramos. <em>Journal of Biogeography, 46</em>(8), 1808-1825.</p>
</div>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = false;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
